bb8/src/inner.rs

```diff
@@ -10,13 +11,36 @@ use tokio::spawn;
 use tokio::time::{interval_at, sleep, timeout, Interval};
 
 use crate::api::{Builder, ConnectionState, ManageConnection, PooledConnection, RunError};
-use crate::internals::{Approval, ApprovalIter, Conn, SharedPool, State};
+use crate::internals::{Approval, ApprovalIter, Conn, SharedPool};
+
+struct SharedPoolInnerStats {
+    gets: AtomicU32,
+    gets_waited: AtomicU32,
```

Comment:
@djc I was thinking that just for the sake to reduce the number of events for having overflows and resets to 0, we could just use `AtomicU64`.

Reply:
Sounds good.